yarn.lock

```diff
@@ -993,7 +993,27 @@ __metadata:
   languageName: node
   linkType: hard
 
-"@metamask/eth-json-rpc-middleware@^15.0.1, @metamask/eth-json-rpc-middleware@workspace:.":
+"@metamask/eth-json-rpc-middleware@npm:^15.0.1":
+  version: 15.3.0
```

Comment:
Hmm, interesting. It looks like we have two versions of this package in the lock. Do you have a guess for why this is the case?

Reply:
I though that was strange too 😄 

Circular dependency via the network controller I believe.